pkg/deps/config/suppliers.go

```diff
@@ -380,16 +380,29 @@ func NewSupplySupplierClientsFn(signingKeyNames []string) SupplierFn {
 			}
 		}
 
+		// Ensure that the gas prices include upokt
+		for _, gasPrice := range gasPrices {
+			if gasPrice.Denom != volatile.DenomuPOKT {
+				// TODO_TECHDEBT(red-0ne): Allow other gas prices denominations once supported (e.g. mPOKT, POKT)
+				// See https://docs.cosmos.network/main/build/architecture/adr-024-coin-metadata#decision
+				return nil, fmt.Errorf("only gas prices with %s denom are supported", volatile.DenomuPOKT)
+			}
+		}
+
+		// Setup the tx client options for the suppliers.
+		txClientOptions := make([]client.TxClientOption, 0)
+		txClientOptions = append(txClientOptions, tx.WithGasPrices(&gasPrices))
+		txClientOptions = append(txClientOptions, tx.WithGasAdjustment(gasAdjustment))
+		txClientOptions = append(txClientOptions, tx.WithGasSetting(&gasSetting))
+		txClientOptions = append(txClientOptions, tx.WithFeeAmount(&feeAmount))
```

Comment:
Just confirming that these are all always populated and/or there's no error if they're not?

Reply:
For the `FeeAmount` is not populated since the `RelayMiner` does not expose that flag.

Removing it from the options.